static/app/components/replays/replayView.tsx

```diff
@@ -8,21 +9,48 @@ import ReplayController from 'sentry/components/replays/replayController';
 import ReplayCurrentUrl from 'sentry/components/replays/replayCurrentUrl';
 import ReplayPlayer from 'sentry/components/replays/replayPlayer';
 
+// How much to reveal under the player, so people can see the 'pagefold' and
+// know that they can scroll the page.
+const BOTTOM_REVEAL_PIXELS = 70;
+
 type Props = {
   isFullscreen: boolean;
   toggleFullscreen: () => void;
 };
 
 function ReplayView({isFullscreen, toggleFullscreen}: Props) {
+  const containerRef = useRef<HTMLDivElement>(null);
+  const playerRef = useRef<HTMLDivElement>(null);
+
+  const [windowInnerHeight, setWindowInnerHeight] = useState(window.innerHeight);
+  const [maxPlayerHeight, setMaxPlayerHeight] = useState(0);
+
+  useEffect(() => {
+    const onResize = debounce(() => {
+      setWindowInnerHeight(window.innerHeight);
+    });
+    window.addEventListener('resize', onResize);
```

Comment:
Should we use `useResizeObserver`?

Reply:
Oh, odd - wonder if it's because of the styled component refs?

Reply:
It's because I wanted to measure something outside of the local component tree. 

The components in this file have have either the previous height value, or are flexible and shrink around that fixed height. What I wanted to measure instead is the added, or removed, pixels and then adjust this component to fit within.

The container size stuff would work with only css grid:`grid-template-rows: auto 1fr auto;` but I need the real `width` and `height` values in javascript in order to set the `transform: scale(?)` attr and fit the replay into our page.. 
I also tried using css variables to set the `scale()`, but `attr(--my-var)` returns a string so the transform wasn't working.